packages/dev/core/src/Misc/index.ts

```diff
@@ -89,3 +89,5 @@ export * from "../ShadersWGSL/rgbdEncode.fragment";
 // CopyTextureToTexture
 export * from "../Shaders/copyTextureToTexture.fragment";
 export * from "../ShadersWGSL/copyTextureToTexture.fragment";
+
+export * from "./tools.internals";

```

Comment:
Revert? Doesn't seem necessary, especially since you are importing directly from the path anyway.

Reply:
Because of UMD? @deltakosh, @sebavan - it seems like we shouldn't export internal tools from the core package, any thoughts?